recipes-graphics/xorg-xserver/xserver-xorg_1.19.1.bbappend

```diff
@@ -0,0 +1,3 @@
+# we don't use this driver; don't build it
+RRECOMMENDS_${PN}_remove = "xf86-input-libinput"
+RRECOMMENDS_${PN}-xwayland_remove = "xf86-input-libinput"
```

Comment:
Why is xf86-input-libinput being removed?  It's running in uivm on my machine?

Why `${PN}-xwayland_remove`?  We shouldn't install xwayland in the first place.

Reply:
Standby, this removal might not be needed after all. We had some UI issues that were fixed with a clean build; this might be one of them. @crogers1 is kicking off a build without this bbappend.

Reply:
During dev we saw some odd build errors that this bbappend fixed.  It seems those may have been from stale build trees and configs, so I don't think this is needed anymore.  Will remove.